src/icons/file/README.md

```diff
@@ -21,107 +21,107 @@ const IconsExample = require('../../../examples/src/IconsExample').default;
 const icons = [
     {
         name: 'IconFileAudio',
-        component: require('../imported/IconFileAudio32').default,
+        component: require('../../icon/content/FileAudio32').default,
```

Comment:
Is this accurate, that using `<IconFileAudio>` would render `<FileAudio32>`?

Reply:
oh good catch, the names should be updated